README.md

```diff
@@ -1,13 +1,123 @@
 # Archive
 
-## Status
+_Acts as a gateway for integrating with Formpipe's proxy, enabling long-term document archiving._
+
+## Getting Started
+
+### Prerequisites
+
+- **Java 21 or higher**
+- **Maven**
+- **Git**
+- **[Dependent services](#dependencies)**
+
+### Installation
+
+1. **Clone the repository:**
+
+   ```bash
+   git clone git@github.com:Sundsvallskommun/api-service-archive.git
+   ```
+
+2. **Configure the application:**
+
+   Before running the application, you need to set up configuration settings.
+   See [Configuration](#Configuration)
+
+   **Note:** Ensure all required configurations are set; otherwise, the application may fail to start.
+
+3. **Ensure dependent services are running:**
+
+   If this microservice depends on other services, make sure they are up and accessible. See [Dependencies](#dependencies) for more details.
+
+4. **Build and run the application:**
+
+     ```bash
+     mvn spring-boot:run
+     ```
+
+## Dependencies
+
+This microservice depends on the following services:
+
+- **Formpipe Proxy**
+
+  - **Purpose:** API used in front of Formpipe LTA's WCF-based API as a REST API, to enable Java-based services and clients to long-term archive documents.
+  - **Additional Information:** Refer to its [product page](https://www.formpipe.com/en/public-sector/software/long-term-archive/) to learn more
+
+Ensure that these services are running and properly configured before starting this microservice.
+
+## API Documentation
+
+Access the API documentation via Swagger UI:
+
+- **Swagger UI:** [http://localhost:8080/api-docs](http://localhost:8080/api-docs)
+
+Alternatively, refer to the `openapi.yml` file located in the project's root directory for the OpenAPI specification.
```

Comment:
Remove?

Reply:
All services have the openapi.yml file available somewhere. Why not refer to this so a potential user can easily access it without starting the service? I update the path. But removing this entirely is counterproductive.

Reply:
Ok, then change text - as it is stated right now it's more of a "do this (point to swagger) or do that". The best case is to both point to swagger (which is done in this readme) and also (if available - which I couldn't see that is was) point to the yaml inside the service aswell.

If both possibilities can be fullfilled, then yes - that's the best path to go.